modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/scala/edu/gemini/seqexec/web/server/common/LogInitialization.scala

```diff
@@ -6,73 +6,77 @@ import java.util.logging.LogManager
 import scala.io.{Codec, Source}
 import scalaz._
 import Scalaz._
-import scalaz.effect.IO
+import scalaz.concurrent.Task
 
-trait LogInitialization {
-  sealed trait LogConfError
-  case class ConfigurationNotFound(s: String) extends LogConfError
-  case class LogDirNotFound(f: File) extends LogConfError
-  case object ConfigurationLoadingError extends LogConfError
+trait AppBaseDir {
+  /**
+    * Calculates the base dir of the application based on the location of "this" class jar file
+    * It will throw an exception if unable to find the base dir
+    */
+  def baseDir: Task[File] = Task.delay {
+    val clazz = this.getClass
+    val fileName = clazz.getResource(s"/${clazz.getName.replace(".", System.getProperty("file.separator"))}.class").getFile
 
-  type LogConf[A] = EitherT[IO, LogConfError, A]
```

Comment:
From a FP stand point it may be easy to disagree with this change, but for the record, I just wanted to say that I think getting getting rid of this type for this case is a great idea. :+1: